lib/l10n/intl_es.arb

```
@@ -399,5 +399,42 @@
   "share": "Compartir",
   "failedToShareInvoice": "Error al compartir factura. Por favor intenta copiarla en su lugar.",
   "openWallet": "ABRIR BILLETERA",
-  "done": "HECHO"
+  "done": "HECHO",
```

RuboCop (1.75.5)

[warning] Duplicated key in hash literal.

(Lint/DuplicateHashKey)